frontend/public/components/overview/pods-overview.tsx

```diff
@@ -135,15 +135,15 @@ export const PodsOverviewContent: React.SFC<PodsOverviewContentProps> = ({
   loadError,
   allPodsLink,
   emptyText,
-  hasBuildConfig,
+  isBuildComplete,
 }) => {
   const {
     metadata: { name, namespace },
   } = obj;
 
   const [showWaitingPods, setShowWaitingPods] = React.useState(false);
   const showWaitingForBuildAlert =
-    hasBuildConfig && isDeploymentGeneratedByWebConsole(obj) && pods.some(isPodWithoutImageId);
+    !isBuildComplete && isDeploymentGeneratedByWebConsole(obj) && pods.some(isPodWithoutImageId);
```

Comment:
This doesn't seem like the right solution as when the build completes this `pods.some(isPodWithoutImageId)` should return `false`. I think you should look into fixing this check and see why this isn't returning `false` when the build completes.